evm_config.yaml

```diff
@@ -0,0 +1,47 @@
+mainnet:
+  rpc_url: "https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}"
+  contract_address: ""
```

Comment:
We should probably name it `pragma_contract_address` to be explicit

Reply:
Done